aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -80,4 +80,21 @@ Tensor to(const Tensor& self, const Tensor& other, bool non_blocking, bool copy)
   return to_impl(self, options, non_blocking);
 }
 
+Tensor to_dense_backward(const Tensor& grad, const Tensor& input_) {
+  AT_ASSERT(input_.layout() != c10::kStrided);
+  if (input_.is_sparse()) {
```

Comment:
for consistency sake, can you change `input._is_sparse()` to check the layout, since that's what you are checking everywhere else.

Reply:
Fixed.